sympy/printing/mathml.py

```diff
@@ -905,7 +920,7 @@ def mathml(expr, printer='content', **settings):
         return MathMLContentPrinter(settings).doprint(expr)
 
 
-def print_mathml(expr, printer='content', **settings):
+def print_mathml(expr, printer='content', symbol_names=None, root_notation=True, order=None):
```

Comment:
It would be very useful if you can add all the settings from #15877 here. Starting with `printer` and `order` (to not break backwards compatibility) and then the rest in alphabetical order.

Even better if they are also added below, but here is the first priority.